src/client/app.tsx

```diff
@@ -30,19 +30,25 @@ export const App = (props: ClientState) => {
     <>
       <Global
         styles={css`
-          ${resets.defaults}
           ${fontFaces}
           html {
             height: 100%;
+            box-sizing: border-box;
           }
           body {
             height: 100%;
+            color: ${neutral[7]};
           }
           #app {
             min-height: 100%;
             display: flex;
             flex-direction: column;
           }
+          *,
+          *:before,
+          *:after {
+            box-sizing: inherit;
+          }
           // Badge is hidden for Gateway, because we're using
           // the legal text to do this job
```

Comment:
I know this wasn't introduced here but though it's worth changing to valid css comments

```suggestion
          /* Badge is hidden for Gateway, because we're using
           the legal text to do this job */
```